Project2015To2017Tests/TestFiles/FileInclusion/SourceFileWithDesigner.Designer.cs

```diff
@@ -1,4 +1,14 @@
-﻿namespace Project2015To2017Tests.TestFiles.FileInclusion {
+﻿//------------------------------------------------------------------------------
+// <auto-generated>
+//     This code was generated by a tool.
+//     Runtime Version:4.0.30319.42000
+//
+//     Changes to this file may cause incorrect behavior and will be lost if
+//     the code is regenerated.
+// </auto-generated>
+//------------------------------------------------------------------------------
+
+namespace Project2015To2017Tests.TestFiles.FileInclusion {
     using System;
     
```

Comment:
I'll get this reverted out and fixed to not happen again, it was put in by mistake

Reply:
decided to leave it in. I can't see an obvious way to stop it happening and it doesn't seem to do any harm

Reply:
Not too worried about the test files, they are as the name implies for test purposes anyway